smart-server/src/test/java/org/smartdata/server/actions/TestActionRegister.java

```diff
@@ -0,0 +1,29 @@
+package org.smartdata.server.actions;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ *
+ */
+public class TestActionRegister {
+
+  @Test
+  public void testNativeClassMap() throws Exception {
+    ActionRegister ar = ActionRegister.getInstance();
+    ar.loadNativeAction();
+    String[] actionNames = ar.namesOfAction();
+    Assert.assertTrue(actionNames.length == 2);
```

Comment:
This assert condition is not stable enough, it fails when actions added or removed